content/tracing/services/resource.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Resource
+kind: Documentation
+further_reading:
+- link: "/tracing/services/service"
+  text: Learn more about services in Datadog
+- link: "/tracing/services/trace"
+  text: "Understand how to read a Datadog Trace"
+---
+
+{{< img src="tracing/services/resource/ressource.png" alt="Ressource" responsive="true" popup="true">}}
+
+## Definition
+
+**A Resource is particular action for a service**.  
+
+* For a web application: some examples might be a canonical URL like `/user/home` or a handler function like `web.user.home` (often referred to as "routes" in MVC frameworks).  
+* For a SQL database: a resource would be the SQL of the query itself like `select * from users where id = ?`.
+
+Resources should be grouped together under a canonical name, like `/user/home` rather than have `/user/home?id=100` and `/user/home?id=200` as separate resources.
+
+These resources can be found after clicking on a particular [service](/tracing/services/#resource).
+
+Resource names: 
+
+* **must be lowercase, alphanumeric characters**
+* **cannot exceed 5000 bytes**
+* **contain spaces** (spaces are replaced with underscores)
+* Must adhere to [metric naming rules](/developers/metrics/).
+
+### Resource Cardinality
+
+When a resource such as URL or SQL query cannot be aggregated, it significantly increases the cardinality of resources (ie. the number of unique aggregate resources) to be stored in Datadog. 
+
+Having a very high cardinality of resources is not useful and becomes difficult to deal with on the Datadog side as each aggregate resources needs to be stored.
+
+As a result we have a hard limit on the cardinality of resources for a given service.
+
+## Out of the box graphs
+
+Datadog provides out of the box graphs for any given Resource:
+
+* Requests - Choose to display:
+    *  The **Total amount of requests** 
+    *  The amount of **Requests per second**
+* Latency -  Choose to display:
+    *  The Avg/p75/p90/p95/p99/Max latency of your traced requests 
+* Error - Choose to display:
+    * The **Total amount of errors** 
+    * The amount of **Errors per second** 
+    * The **% Error Rate** 
+* Sub-Services: When there is multiple service involved a 4th graph is available that breaks down your **Total time spent**/**%of time spent**/**Avg time per request** of your service by *services* or *type*.
```

Comment:
Same comment as done for the service section below.

Reply:
👌